### PR TITLE
Model

### PR DESCRIPTION
**Pull request by Grogro21**
Believe in your dreams

### CODE REVIEW

File: AppFormes/ShapeSMOdel/src/Shape.cs [hunk 1]
```diff
@@ -15,6 +15,14 @@ public Shape(string name, Point position, string theme)
             _theme = theme;
         }
 
+        public Shape(SelectableShape selectableShape, Point realPosition)
```
Comment (Fuzzy2319) [inline AppFormes/ShapeSMOdel/src/Shape.cs]:
```cs
public Shape(SelectableShape selectableShape, Point realPosition): this(selectableShape.GetName(), selectableShape.GetTheme(), realPosition){}
```

File: AppFormes/ShapeSMOdel/src/SelectableShape.cs [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
-            _positionAnswer = position;
             _isSelected = false;
         }
 
-        public bool Selected { get => _isSelected; set => _isSelected = value; }
+        public bool IsSelected { get => _isSelected; set => _isSelected = value; }
+        public Shape Shape { get => _shape; }
 
-        public string getName()
+        public string GetName()
```
Comment (Fuzzy2319) [inline AppFormes/ShapeSMOdel/src/SelectableShape.cs:17]: property

File: AppFormes/ShapeSMOdel/src/SelectableShape.cs [hunk 3]
```diff
@@ -1,26 +1,28 @@
...
         {
             return _shape.Name;
         }
+
+        public string GetTheme()
```
Comment (Fuzzy2319) [inline AppFormes/ShapeSMOdel/src/SelectableShape.cs:22]: property

File: AppFormes/ShapeSMOdel/src/Utilities.cs [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            for (int i = 0; i < number; i++)
+            {
+                do
+                {
+                    Random rand = new Random();
+                    themeToAdd = themes[rand.Next(themes.Length)];
+                }
+                while (themes.Contains(themeToAdd));
+            }
+            return themes;
```
Comment (Fuzzy2319) [inline AppFormes/ShapeSMOdel/src/Utilities.cs:73]: empty line before return

**Review by Fuzzy2319 — CHANGES_REQUESTED** [review]

File: AppFormes/ShapeSMOdel/src/SelectableShape.cs (hunk 3 above)
Comment (Grogro21) [reply AppFormes/ShapeSMOdel/src/SelectableShape.cs:22]: osef

File: AppFormes/ShapeSMOdel/src/Utilities.cs (hunk 4 above)
Comment (Grogro21) [reply AppFormes/ShapeSMOdel/src/Utilities.cs:73]: osef

File: AppFormes/ShapeSMOdel/src/Shape.cs (hunk 1 above)
Comment (Grogro21) [reply AppFormes/ShapeSMOdel/src/Shape.cs]: ok

**Review by Fuzzy2319 — APPROVED** [review]

**Pull request merged**